pkg/cache/multi_namespace_cache.go

```diff
@@ -146,11 +142,14 @@ func (c *multiNamespaceCache) GetInformerForKind(ctx context.Context, gvk schema
 
 func (c *multiNamespaceCache) Start(ctx context.Context) error {
 	// start global cache
-	go func() {
-		if err := c.clusterCache.Start(ctx); err != nil {
-			log.Error(err, "multi-namespace cache failed to start cluster scoped cache")
-		}
-	}()
+	if c.clusterCache != nil {
```

Comment:
Do we have to check this in more places to avoid panics if folks misconfigure the cache?

If I see correctly we should never hit those cases with the way we delegate / fallback, right?

Reply:
Yeah, there is no way to access the multiNamespace cache directly as it is not exported and we do have testcases in cache_test.go where this is used only for a namespaces type and thus doesn't have a clusterCache. That is how I found out that these are needed :)

Reply:
Perfect!